explainer.html

```diff
@@ -91,34 +97,34 @@
     <h2>Terminology</h2>
 
     <p>
-For a precise definition for the various terms and notions, the reader should
-refer to the formal RDF specification [[rdf11-concepts]]. Note also that the
-term “Linked Data” is used as a synonym to “RDF Datasets”.
+For a precise definition of the various terms and concepts, the reader should
+refer to the formal RDF specification [[rdf11-concepts]].
     </p>
 
     <dl>
       <dt id='dataset'>RDF Datasets</dt>
       <dd>
         <p>
-<span class='rdf'>R</span>, <span class='rdf'>S</span>,&nbsp; etc., denote an <a
+<span class='rdf'>R</span>, <span class='rdf'>R'</span> and <span class='rdf'>S</span> each denote an <a
 href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Dataset</a>
 [[rdf11-concepts]].
         </p>
-        <p>
-The term “named graphs” is also used in literature or in applications.
-        </p>
       </dd>
 
       <dt id='identical'>Identical RDF Datasets</dt>
       <dd>
         <p>
-<span class='rdf'>R = S</span> denotes two <em>identical</em> RDF Datasets.
+<span class='rdf'>R = S</span> denotes that <span class='rdf'>R</span> and <span class='rdf'>S</span> are <em>identical</em> RDF Datasets. 
```

Comment:
Shouldn't the same modification be applied on the section about "isomorphism"?

Reply:
Done in latest commit; merging